cli/tests/e2e/snapshots/test_semgrep_core_parse_error/test_file_parser__failure__error_messages/settings1/out.json

```diff
@@ -3,13 +3,13 @@
     {
       "code": 3,
       "level": "warn",
-      "message": "Syntax error at line targets/bad/invalid_python.py:1:\n `def foo(a)\n    tree = ElementTree.parse('country_data.xml')` was unexpected",
+      "message": "Syntax error at line targets/bad/invalid_python.py:1:\n `def foo(a)\n    tree = ElementTree.parse('country_data.xml')\n    root = tree.getroot()` was unexpected",
       "path": "targets/bad/invalid_python.py",
       "spans": [
         {
           "end": {
-            "col": 49,
-            "line": 2
+            "col": 26,
```

Comment:
why those changes? How adding contributions in the scan result has an impact on the matching position?

Reply:
I believe this was from my running the tests locally with python3.11. Reverted these snapshot changes